### ISSUE TITLE
Add examples with ideal component documentation

### ISSUE THREAD

**Issue by alexprey** [alexprey/sveltedoc-parser#25]
Add few examples of what document structure can be parsed by this package

**Comment by TheComputerM** [comment]
Shall I make a basic template? I do not know the full extent of sveltedoc-parser.

**Comment by TheComputerM** [comment]
[Example FIle](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/svelte-materialify/src/components/Button/Button.svelte)
[Output](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/api-generator/src/Button.json)

-------------------

[Example File](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/svelte-materialify/src/components/Alert/Alert.svelte)
[Output](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/api-generator/src/Alert.json)

**Comment by alexprey** [comment]
@TheComputerM Thanks!

One small note for your component docs
```
@type {button|reset|submit}
```
This statement means that you have types `button`, `reset`, `submit`. However, you pass the string values for this prop, so you should write comment in the following format (just provide commas):
```
@type {'button'|'reset'|'submit'}
```
And then in output you should have following:
```
"type": [
          {
            "kind": "type",
            "text": "x-small",
            "type": "string"
          },
          {
            "kind": "type",
            "text": "small",
            "type": "string"
          },
```

**Comment by TheComputerM** [comment]
When I use strings like you say, I get the output:
```
"type": [
          {
            "kind": "type",
            "text": "'x-small'",
            "type": "string"
          },
          {
            "kind": "type",
            "text": "'small'",
            "type": "string"
          },
```

**Comment by alexprey** [comment]
Yes, this format is a correct, I'm forgot about quotes in text property

**Comment by alexprey** [comment]
In [this commit](https://github.com/alexprey/sveltedoc-parser/commit/6726d213113d1c70f0b268ae94e66919e3011c39) I'm prepare base for writing examples of components and parsed document structure.

**Comment by TheComputerM** [comment]
Nice
